cmd/cmd.go

```diff
@@ -1193,49 +1182,20 @@ func output(val interface{}) {
 	fmt.Println(val)
 }
 
-// Get the base repo path for the user, will create it if missing
-// Unless provided, it defaults to ~/.textile/repo
-func getBaseRepo(baseRepo string) (string, error) {
-	if len(baseRepo) == 0 {
-		// get homedir
-		home, err := homedir.Dir()
-		if err != nil {
-			return "", fmt.Errorf("get homedir failed: %s", err)
-		}
-
-		// ensure app folder is created
-		appDir := filepath.Join(home, ".textile")
-		if err := os.MkdirAll(appDir, 0755); err != nil {
-			return "", fmt.Errorf("create repo directory failed: %s", err)
-		}
-		baseRepo = filepath.Join(appDir, "repo")
+// Get the full repo path for the user, will create it if missing
+func getRepo(repo string, baseRepo string, accountAddress string) (string, error) {
+	var finalRepo string
+	if len(repo) > 0 {
+		finalRepo = repo
+	} else if len(baseRepo) > 0 && len(accountAddress) > 0 {
+		finalRepo = path.Join(baseRepo, accountAddress)
+	} else {
+		return "", fmt.Errorf("you must specify --repo or --base-repo and --account-address flags")
 	}
-	return baseRepo, nil
-}
-
-// Get the full repo path for the user, will use the single
-// directory inside baseRepo if accountAddress isn't provided
-func getRepo(baseRepo string, accountAddress string) (string, error) {
-	if len(accountAddress) == 0 {
-		files, err := ioutil.ReadDir(baseRepo)
-		if err != nil {
-			return "", err
-		}
-		if len(files) == 0 {
-			return "", fmt.Errorf("no account repos initialized in: %s", baseRepo)
-		}
-		if len(files) > 1 {
-			for _, file := range files {
-				if !file.IsDir() && file.Name() == "textile" {
-					// This is a pre 0.7.2 repo, not subdir under account path
-					return baseRepo, nil
-				}
-			}
-			return "", fmt.Errorf("there are multiple accounts initialzed in %s, you need to specify account-address", baseRepo)
-		}
-		return path.Join(baseRepo, files[0].Name()), nil
+	if err := os.MkdirAll(finalRepo, 0755); err != nil {
```

Comment:
In the case where you have an existing repo folder and are not using a base repo, this may result in the same docker issue we saw with doodle. It would be nice to check this behavior.

Reply:
ok, I'll try to test it out.

Reply:
I ended up just removing all the code in `cmd.go` that attempts to create directories... it's not needed because all the directory creation is handled elsewhere in the initialization process. I'll still make sure this works with the Dockerfiles

Reply:
Ok, seems to run fine in docker. I ran these commands to test:
```
docker build --file=./Dockerfile --tag=textile .
docker run textile
docker build --file=./Dockerfile.cafe --tag=textile-cafe .
docker run textile-cafe
```